1.9/usage/marathon/application-basics.md

```diff
@@ -0,0 +1,143 @@
+
+* `.tgz`
+* `.tar.gz`
+* `.tbz2`
+* `.tar.bz2`
+* `.txz`
+* `.tar.xz`
+* `.zip`
+
+The following example shows you how this looks in practice. Assume you have an application executable in a zip file at `https://example.com/app.zip`. This zip file contains the script `cool-script.sh`, which you want to execute. Here's how:
+
+```json
+{
+    "id": "basic-2", 
+    "cmd": "app/cool-script.sh",
+    "cpus": 0.1,
+    "mem": 10.0,
+    "instances": 1,
+    "uris": [
+        "https://example.com/app.zip"
+    ]
+}
+```
+
+In contrast to the example `basic-1` we now have a `cmd` with the value `app/cool-script.sh`. When the zip file gets downloaded and extracted, a directory `app` according to the file name `app.zip` is created and the content of the zip file is extracted into it.
+
+You can specify more than one resource. For example, you could provide a git repository and some resources from a CDN as follows:
+
+```json
+{
+    "uris": [
+        "https://git.example.com/repo-app.zip", "https://cdn.example.net/my-file.jpg", "https://cdn.example.net/my-other-file.css"
+    ]
+}
+```
+
+A typical pattern in the development and deployment cycle is to have your automated build system place the app binary in a location that's downloadable via an URI. Marathon can download resources from a number of sources. Marathon supports the following [URI schemes](http://tools.ietf.org/html/rfc3986#section-3.1):
+
+* `file:`
+* `http:`
+* `https:`
+* `ftp:`
+* `ftps:`
+* `hdfs:`
+* `s3:`
+* `s3a:`
+* `s3n:`
+
+## A Simple Docker-based Application
+
+With Marathon it is straightforward to run applications that use Docker images.
+
+In the following example application definition, we will focus on a simple Docker app: a Python-based web server using the image [python:3](https://registry.hub.docker.com/_/python/). Inside the container, the web server runs on port `8080` (the value of `containerPort`). `hostPort` is set to `0` so that Marathon assigns a random port on the Mesos agent, which is mapped to port 8080 inside the container.
+
+```json
+{
+  "id": "basic-3",
+  "cmd": "python3 -m http.server 8080",
+  "cpus": 0.5,
+  "mem": 32.0,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "python:3",
+      "network": "BRIDGE",
+      "portMappings": [
+        { "containerPort": 8080, "hostPort": 0 }
+      ]
+    }
+  }
+}
+```
+
+In this example, we are going to use the Marathon REST API to deploy the app `basic-3` from the DC/OS CLI:
+
+```sh
+ curl -H "Authorization: token=$(dcos config show core.dcos_acs_token) -X POST <dcos-url>/service/marathon/v2/apps -d @basic-3.json -H "Content-type: application/json"
+```
+
+When you submit the above application definition to DC/OS, you should see your new service running from the **Services** tab of the DC/OS GUI.
+
```

Comment:
@pyronicide I've ported this document from the Marathon site and modified it for DC/OS. The last piece of the puzzle is this final instruction -- do you happen to know where exactly a user should go to see the contents of the container's root directory?